interface/patient_file/summary/demographics.php

```diff
@@ -473,120 +476,205 @@ function setMyPatient() {
 </head>
 
 <body class="body_top patient-demographics">
-
```

Comment:
unless of course these are all code changes or just simple white spacing changes (since can visualize diff without whitespaces via commandline).